signals.go

```diff
@@ -112,9 +112,8 @@ func (h *signalHandler) forward(process *libcontainer.Process, tty *tty, detach
 			// and it should not be forwarded to the container.
 			// Do nothing.
 		default:
-			us := s.(unix.Signal)
-			logrus.Debugf("forwarding signal %d (%s) to %d", int(us), unix.SignalName(us), pid1)
-			if err := unix.Kill(pid1, us); err != nil {
+			logrus.Debugf("forwarding signal %s to container", s)
```

Comment:
LGTM.

In fact, we can remove the signal num in the debug log, but I think the pid is useful for debug.
Could you mind add it back? For example:
```
logrus.Debugf("forwarding signal %s to container(pid: %d)", s, pid1)
```
Thanks.

Reply:
I've reverted the logging changes (as `os.Signal.String()` gives some garbage like "interrupt")